### ISSUE TITLE
VS Code: Expose WordPress debug.log

### ISSUE THREAD

**Issue by jonathanbossenger** [WordPress/playground-tools#27]
When developing in VS Code using the extension, it would be ideal to have the WordPress debug.log enabled, and logging to some arbitrary file in the plugin or theme directory.

The suggested wp-config.php settings would be:

```
define( 'WP_DEBUG', true ); // enable debugging
define( 'WP_DEBUG_DISPLAY', false ); // prevent errors/warnings from displaying on screen
define( 'WP_DEBUG_LOG', '/path/to/active/folder/debug.log' ); // log to errors/warnings to the debug.log file in the open folder
```

Bonus points if it could be possible to enable/disable the WP_DEBUG_DISPLAY constant, in case a developer wanted to do so.

**Comment by adamziel** [comment]
Great idea. VS Code has the "terminals" in the bottom bar, one of them could say `WordPress debug` and display debug information.

**Comment by adamziel** [comment]
Cc @danielbachhuber @sejas for wp-now

**Comment by adamziel** [comment]
cc @dmsnell – you may find this one interesting and I know it was one thing blocking @jonathanbossenger from promoting the vs code extension to a wider audience

**Comment by dmsnell** [comment]
yeah definitely. I think having a debug panel sounds like a great idea.
